test-suites/cvehound

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import json
+from optparse import OptionParser
+
+def convert(bmeta, report):
+    rev = bmeta['revision']
+    env = bmeta['environment']
+    kernel = bmeta['kernel']
+    kernelci = {
+        'job': 'mainline',
+        'git_branch': rev['branch'],
+        'git_commit': rev['commit'],
+        'kernel': rev['describe_verbose'],
+
+        'build_environment': env['name'],
+        'arch': env['arch'],
+        'defconfig': kernel['defconfig'],
+        'defconfig_full': kernel['defconfig_full'],
+
+        'lab_name': 'manual',
+        'device_type': 'shell',
+
+        'name': 'cvehound',
+    }
+
+    cves = {}
+    for cve in report['args']['cve']:
+        cves[cve] = 'PASS'
+    for cve in report['results']:
+        cves[cve] = 'FAIL'
+    if cves:
+        kernelci['test_cases'] = []
+        for name, status in cves.items():
+            kernelci['test_cases'].append({'name': name, 'status': status})
+    return kernelci
+
+def main():
+    parser = OptionParser()
+    parser.add_option('-b', '--bmeta',
+                      dest="bmeta", help="kernelci's bmeta", metavar="FILE")
+    parser.add_option('-r', '--report',
+                      dest="report", help="cvehound report", metavar="FILE")
+    parser.add_option('-o', '--output',
+                      dest="output", help="cvehound report suitable for kernelci", metavar="FILE")
+    (options, args) = parser.parse_args()
+    if args:
+        print("Unknown arguments:", args, file=sys.stderr)
+        exit(1)
+
+    if not options.bmeta:
+        print("Please, specify path to kernelci's bmeta with --bmeta", file=sys.stderr)
+        exit(2)
+    if not options.report:
+        print("Please, specify path to cvehound's report with --report", file=sys.stderr)
+        exit(2)
+    if not options.output:
+        print("Please, specify --output", file=sys.stderr)
+        exit(2)
+
+    if not os.path.isfile(options.bmeta):
+        print("Can't find file", options.bmeta, file=sys.stderr)
+        exit(3)
+    if not os.path.isfile(options.report):
+        print("Can't find file", options.report, file=sys.stderr)
+        exit(3)
+    if os.path.isfile(options.output):
+        print("File", options.output, "already exists", file=sys.stderr)
+        exit(3)
+
+    with open(options.bmeta, 'rt') as fh:
```

Comment:
This will need to be reworked when we start running the tests automatically.  The meta-data (revision etc.) will be passed to the job somehow, but probably not via a bmeta.json file.  I think this is OK for manual experiments so we can merge it as-is for now.